misc/formatter/modules/formatter-core/src/main/java/org/ballerinalang/formatter/core/FormattingOptions.java

```diff
@@ -148,6 +182,26 @@ public ImportFormattingOptions getImportFormattingOptions() {
         return importFormattingOptions;
     }
 
+    public FunctionFormattingOptions getFunctionFormattingOptions() {
+        return functionFormattingOptions;
+    }
+
+    public BlockFormattingOptions getBlockFormattingOptions() {
+        return blockFormattingOptions;
+    }
+
+    public boolean isRecordBraceSpacing() {
+        return recordBraceSpacing;
+    }
+
+    public boolean alignConsecutiveDefinitions() {
+        return alignConsecutiveDefinitions;
+    }
+
+    public boolean alignMultiLineQueries() {
+        return alignMultiLineQueries;
+    }
```

Comment:
Function names are not consistent since the first two functions have `get` prefixed to them